### ISSUE TITLE
Training issues about new version of Chiron

### ISSUE THREAD

**Issue by qianlou** [haotianteng/Chiron#53]
Hi Teng,

Thanks for your previous advice and updates. I used wget command to download your data.genomicsresearch.org/Projects/basecall/Human_CHR19/file_batch data. After that, I used "python chiron/chiron_train.py  -i /home/louqian/Chiron/traindata/ -o /home/louqian/Chiron/trainlog/ -m train_test
" to train my new model. But there is a error and  I attached the output on error1.txt. 
[error1.txt](https://github.com/haotianteng/Chiron/files/2040990/error1.txt). I found that the definition of inference function in  chiron_model file has five parameters, but  there are 4 parameters in chiron_train file when calls this inference function. Maybe this is a bug. I also tried another trainning method which  uses chiron_rcnn_train.py. I used chiron/utils/raw.py to generate tf.records , signal and label data. But I can't still train it. Can you try these two methods again? Any advice will be appreciated. 

Sincerely,
Qian

**Comment by haotianteng** [comment]
Can you git pull to update Chiron to the latest version?

**Comment by qianlou** [comment]
Thanks for your advice, Teng. The latest version deals with my last question. But now when I run it, the console prints "model init finished, begin trainning". Then after 30 minutes, there is no other reactions and prints. So did you see this question?

Sincerely, 
Qian

**Comment by haotianteng** [comment]
No I didn't encounter this problem, so the model geaph has been created
successfully if you see the message, it runs fine on my PC, what is your
hardware setting?

**Comment by haotianteng** [comment]
The model will always contain one or several CNN layers, set rnn_layer=0
will make the model only use CNN, getcnnlogits function is used to transfer
the CNN output directly to a logits. If rnn layer>0, the getcnnfeature
function will be called instead, and will pass the CNN output to RNN.

For model construction, please take a look at the chiron_model.py module,
and also the cnn.py and rnn.py module.

Thank you
Teng

**Comment by haotianteng** [comment]
The model will always contain one or several CNN layers, set rnn_layer=0 will make the model only use CNN, getcnnlogits function is used to transfer the CNN output directly to a logits. If rnn layer>0, the getcnnfeature function will be called instead, and will pass the CNN output to RNN.

For model construction, please take a look at the chiron_model.py module, and also the cnn.py and rnn.py module.

Thank you
Teng

**Comment by qianlou** [comment]
Thanks for your reply,Teng. I have read all chrion_model.py, cnn.py and rnn.py. I know the getcnnfeature() function will be called whenever rnn layer>0 or not. But from my opinion, getcnnfeature() function does't include any operations such as matrix multiply or convolution operations just some reshape operations.  Can getcnnfeature() function get CNN output from signal input? 
Sincerely,
Qian
